website/docs/pact_broker/webhooks/template_library.md

```diff
@@ -480,3 +428,111 @@ The following example shows how to trigger a Drone CI build with parameters.
 Ref:
 
 * [Drone API - Build Create](https://readme.drone.io/api/builds/build_create/).
+
+## Triggering Notifications
+
+### Commit Status
+
+#### Github - publish commit status
+
+Report the pact verification status back to the consumer project in Github.
+
+##### CLI
+
+```
+pact-broker create-webhook \
+  'https://api.github.com/repos/<organization>/<project>/statuses/${pactbroker.consumerVersionNumber}' \
+  -X POST \
+  -H "Content-Type: application/json" \
+  -d '{ "state": "${pactbroker.githubVerificationStatus}", "description": "Pact Verification Tests ${pactbroker.providerVersionTags}", "context": "${pactbroker.providerName}", "target_url": "${pactbroker.verificationResultUrl}" }' \
+  --user username:password \
+  --description "Publish pact verification status to Github" \
+  --contract-published \
+  --provider-verification-published \
+  --broker-base-url <your-broker>
+```
+
+##### JSON
+
+```javascript
+{
+  "consumer": {
+    "name": "<consumer name>"
+  },
+  "events": [
+    {
+      "name": "contract_published"
+    },
+    {
+      "name": "provider_verification_published"
+    }
+  ],
+  "request": {
+    "method": "POST",
+    "url": "https://api.github.com/repos/<organization>/<project>/statuses/${pactbroker.consumerVersionNumber}",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "body": {
+      "state": "${pactbroker.githubVerificationStatus}",
+      "description": "Pact Verification Tests ${pactbroker.providerVersionTags}",
+      "context": "${pactbroker.providerName}",
+      "target_url": "${pactbroker.verificationResultUrl}"
+    },
+    "username": "USERNAME",
+    "password": "PASSWORD"
+  }
+}
+```
+
+### Collaboration Platforms
+
+#### Slack - post notification - contract_published
+
+```javascript
+{
+  "events": [
+    {
+      "name": "contract_published"
+    }
+  ],
+  "request": {
+    "method": "POST",
+    "url": "https://hooks.slack.com/services/<webhook id>",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "body": {
+      "channel": "#<your-channel-here>",
+      "username": "webhookbot",
+      "text": "New version of pact created for ${pactbroker.consumerName}/${pactbroker.providerName}: <${pactbroker.pactUrl}|contract-details>",
+      "icon_emoji": ":ghost:"
+    }
+  }
+}
+```
+
+#### Slack - post notification - provider_verification_published
+
+```javascript
+{
+  "events": [
+    {
+      "name": "provider_verification_published"
+    }
+  ],
+  "request": {
+    "method": "POST",
+    "url": "https://hooks.slack.com/services/<webhook id>",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "body": {
+      "channel": "#<your-channel-here>",
+      "username": "webhookbot",
+      "text": "Pact Verification published for $${pactbroker.consumerName}/$${pactbroker.providerName}<$${pactbroker.verificationResultUrl}|$${pactbroker.githubVerificationStatus}>",
```

Comment:
I don't think we need the double `$$` here @YOU54F. We only needed that in the Makefile to stop the Makefile thinking that the webhook template parameter was a bash parameter.